src/Symfony/Bundle/DebugBundle/composer.json

```
@@ -1,7 +1,7 @@
 {
     "name": "symfony/debug-bundle",
     "type": "symfony-bundle",
-    "description": "Provides a tight integration of the Symfony Debug component into the Symfony full-stack framework",
+    "description": "DebugBundle provides a tight integration of the Symfony VarDumper component and the ServerLogCommand from MonologBridge into the Symfony full-stack framework",
```

I'd keep the previous wording to be consistent with other components & bundles descriptions:

https://github.com/symfony/symfony/blob/474fc8112bf0e4bc0fc3353b30b3c36361e3d8b1/src/Symfony/Bundle/WebProfilerBundle/composer.json#L4

https://github.com/symfony/symfony/blob/474fc8112bf0e4bc0fc3353b30b3c36361e3d8b1/src/Symfony/Bundle/TwigBundle/composer.json#L4

https://github.com/symfony/symfony/blob/6.1/src/Symfony/Bundle/SecurityBundle/composer.json#L4 

```suggestion
    "description": "Provides a tight integration of the Symfony VarDumper component and the ServerLogCommand from MonologBridge into the Symfony full-stack framework",
```

---

My bad indeed (copy pasted :sweat_smile:)